src/init.cpp

```diff
@@ -1523,18 +1529,40 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
         }
     }
 
+    // block tree db settings
```

Comment:
We should add some help lines for these new options. Moreover, they are used only for block tree db (not coin db), therefore we should probably change the name of this constant.